odata-url-conventions/4 Resource Path.md

```diff
@@ -373,7 +382,9 @@ http://host/service/Employees('A1245')
 
 Services MAY support an alternate convention for addressing entities by
 appending a segment containing the unprefixed and unquoted key value to the URL of the
-collection containing the entity. Forward-slashes in key value segments
+collection containing the entity. If the declared entity type of the collection does not 
+define a key (for example, Edm.EntityType), then a cast segment to a type defining a key 
+MUST follow the collection, before appending the key value segment. Forward-slashes in key value segments
```

Comment:
This is a potentially breaking change, existing services could in 4.01 allow key-as-segment without a type-cast segment.

Reply:
A type-cast segment should be valid for key-as-segment as well.